dao/src/main/java/se/fortnox/reactivewizard/db/Named.java

```diff
@@ -0,0 +1,12 @@
+package se.fortnox.reactivewizard.db;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.PARAMETER)
+public @interface Named {
```

Comment:
Perhaps we should remove this and use javax.binding.Named, or simply just require that you use compiler option "-paramaters"

Reply:
Are there any use cases for @Named that we lose by dropping it and going with -parameters?

Reply:
Code-wise it looks like we'd get rid of a few lines of extra logic and if everyone can use -parameters they will get cleaner DAO:s as well.

Reply:
Funny that you accidentally referred to a github user there ;-) Maybe we get our first outsider!

I don't think we loose anything, but I believe there might be a migration for those who have a conflict between the value in the annotation and the name of the parameter. But if the annotation is removed, they will get a compilation error so that should be obvious.

Yolo!

Reply:
We would lose some flexibility and force all projects to use -parameters. But I think it's acceptable to remove it - we can always add it again should the need arise.